src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor

```diff
@@ -16,9 +16,15 @@
 
 <PageTitle><ApplicationName ResourceName="@nameof(Dashboard.Resources.StructuredLogs.StructuredLogsPageTitle)" Loc="@Loc" /></PageTitle>
 
-<div class="logs-layout">
-    <h1 class="page-header">@Loc[nameof(Dashboard.Resources.StructuredLogs.StructuredLogsHeader)]</h1>
-    <FluentToolbar Orientation="Orientation.Horizontal">
+<AspirePageContentLayout
```

Comment:
The bottom padding/margin is quite large and causes the scroll bar to appear when it looks like it doesn't need to:

![image](https://github.com/dotnet/aspire/assets/303201/0ec006c6-a2f0-4a7b-8b28-7d60a7822440)

Reply:
The padding/margin comes from fluentui-blazor. I checked this dialog and the only case where this appears is on very low height, not even on mobile devices with 100% viewport sizes. ie:

![image](https://github.com/dotnet/aspire/assets/20359921/bfa2c46a-ec7a-44f1-8de1-d815d0544b1a)
![image](https://github.com/dotnet/aspire/assets/20359921/fcc1ae88-5165-4276-a4db-b5207f2a8894)

This does not seem to be an issue.